aten/src/ATen/native/metal/MetalPrepackOpRegister.cpp

```diff
@@ -3,7 +3,7 @@
 #include <ATen/ATen.h>
 
 
-#if defined(C10_IOS)
+#if (C10_IOS || TARGET_OS_MAC)
```

Comment:
Nothing wrong about this - just feel that having C10_IOS (defined by us) and TARGET_OS_MAC (defined by apple) on the same condition is a little unbalanced, lol.

If this condition is going to be used at other places, shall we create a dedicated macro? Not sure what's the best convention in PyTorch codebase (cc: @dzhulgakov):
- if it's targeting apple devices, shall we introduce 'C10_APPLE'?
- or, can we simply use some master feature switch? e.g. '#if AT_METAL_ENABLED()' seems to be one of the common pattern.

Reply:
How about (TARGET_OS_PHONE || TARGET_OS_MAC). Would that make you feel better 🤣